lerna.json

```diff
@@ -7,7 +7,9 @@
     "modules/*"
   ],
   "command": {
-    "publish": {},
+    "publish": {
+      "ignore": "modules/jupyter-widget/*",
```

Comment:
I don't think this is a valid field. https://github.com/lerna/lerna#lernajson

I'd say add `ignoreChanges`, then set `private: true` in jupyter-widget's package.json